inspirehep/modules/authors/views.py

```diff
@@ -290,11 +291,13 @@ def submitnew():
     visitor = DataExporter()
     visitor.visit(form)
 
-    myobj = DbWorkflowObject.create_object(id_user=current_user.get_id())
-    myobj.set_data(visitor.data)
+    workflow_object = DbWorkflowObject.create_object(id_user=current_user.get_id())
+    workflow_object.data = visitor.data
+    workflow_object.save()
+    db.session.commit()
```

Comment:
I don't think it is strictly required as inside the create_object and save calls there are nested transactions and as this is at top-level before the request ends we need the specific commit call.